### DISCUSSION TITLE
Can Quartz.Net tell the next running schedule?

### DISCUSSION THREAD

**Discussion by advapiIT** [quartznet/quartznet#957] [General] [answered]
Hello again,
is it possible to have next run occurrences from a specified job directly via Quartz.Net or have I to parse the cron expression and ad to the DateTime.Now?

Thanks

**Comment by lahma** [comment, marked as answer]
Cross-posting is a really bad practice.

https://stackoverflow.com/questions/63826746/show-next-job-execution-scheduled-time
